test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsComprehensive.g.cs

```diff
@@ -73,6 +73,7 @@ public class PackageVersionsComprehensive
                 new object[] { "2.12.0" },
                 new object[] { "2.12.1" },
                 new object[] { "2.12.2" },
+                new object[] { "2.12.3" },
```

Comment:
The changes in this file are unrelated to this new instrumentation. Can you leave them out of this PR? (feel free to push a separate PR to update these)

Reply:
reverted them:)